Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-problems.tsx

```diff
@@ -11,6 +15,7 @@ interface IProblemsContext {
     };
     actions: {
         selectProblemById: (id: number) => void;
+        getProblemResourceFile: (resourceId: number) => Promise<void>;
```

Comment:
IMO, `downloadProblemResourceFile` is better name. `get...` implies a return value